src/modules/todo.js

```diff
@@ -0,0 +1,121 @@
+const taskContainer = document.querySelector('.task-container');
+
+export default class MyTodo {
+  constructor() {
+    this.tasks = [];
+  }
+
+    addTask = (description) => {
+      const myTask = {
+        description,
+        completed: false,
+        index: this.tasks.length + 1,
+      };
+      this.tasks.push(myTask);
+      this.setStorage();
+    }
+
+    remove = (id) => {
+      this.resetIndex();
+      this.tasks.forEach((task) => {
+        if (task.index === Number(id)) {
+          this.tasks
+            .splice(task.index - 1, 1);
+          this.resetIndex();// Update of indexes after deleting
+        }
+      });
+    }
+
+    resetIndex = () => {
+      let initialIndex = 1;
+      this.tasks.forEach((task) => {
+        task.index = initialIndex;
+        initialIndex += 1;
+      });
+    }
+
+    cleanCompleted = () => {
+      this.tasks = this.tasks.filter((task) => task.completed === false);
+      this.setStorage();
+    }
+
+    taskTemplate = (task) => `
+        <div class="task">
+            <div class="t-left" id=${task.index}>
+                <input 
+                    type="checkbox" 
+                    name="Drag" 
+                    class='checkbox' ${task.completed ? 'checked' : ''} 
+                    id=${task.index}> 
+                <input 
+                    type="text" 
+                    id="${task.index}" 
+                    class=${task.completed ? 'line task-label' : 'task-label'} 
+                    value="${task.description}">
+            </div>
+            <p class="deleteTask" id=${task.index}>Delete</p> 
+        </div>`;
+
+    displayTasks = () => {
+      this.getStorage();
+      taskContainer.innerHTML = '';
+      this.tasks
+        .forEach((task) => taskContainer.insertAdjacentHTML('beforeend', this.taskTemplate(task)));
+      const checkboxes = document.querySelectorAll('.checkbox');
+      const taskLabel = taskContainer.querySelectorAll('.task-label');
```

Comment:
Kindly put declared variables at the top of the script or top part in the function